Makefile

```diff
@@ -17,7 +17,14 @@ certs:
 install: certs
 	kubectl create ns $(NAMESPACE) || true
 	kubectl config set-context --current --namespace $(NAMESPACE)
-	helm install connaisseur helm --wait
+	#
+	#=============================================
+	#
+	# The installation may last up to 5 minutes.
+	#
+	#=============================================
+	#
+	helm install connaisseur helm --atomic
```

Comment:
I like the idea of the `--atomic`, but since there's resources like the webhook, which are not directly deployed, would it be advantageous to have something along the lines of `helm instal || make uninstall` instead (or additionally), which would also take care of indirectly created resources?

Reply:
I don't have a simple solution for that in mind and I think that adding `--atomic` already improves the installation process in some cases as you can run `make install` even if a previous `make install` failed.